#SAK-12364 
=svnm= https://source.sakaiproject.org/svn/assignment/trunk@39609 assignment

#Fill in the SAK or something unique, leave it like this if theres no real patch
--- /dev/null
+++ SAK-12364.tmp
@@ -0,0 +1 @@
+ 
